wolfcrypt/src/random.c

```diff
@@ -1671,20 +1678,27 @@ static int _InitRng(WC_RNG* rng, byte* nonce, word32 nonceSz,
                     ret = DRBG_FAILURE;
                 }
             }
-#else
+    #else
             ret = wc_GenerateSeed(&rng->seed, seed, seedSz);
-#endif
+    #endif
             if (ret == 0)
                 ret = wc_RNG_TestSeed(seed, seedSz);
             else {
+    #if defined(DEBUG_WOLFSSL)
+                WOLFSSL_MSG_EX("wc_RNG_TestSeed failed... %d", ret);
+    #endif
                 ret = DRBG_FAILURE;
                 rng->status = DRBG_FAILED;
             }
 
             if (ret == DRBG_SUCCESS)
                 ret = Hash_DRBG_Instantiate((DRBG_internal *)rng->drbg,
-                            seed + SEED_BLOCK_SZ, seedSz - SEED_BLOCK_SZ,
-                            nonce, nonceSz, rng->heap, devId);
+                    seed + SEED_BLOCK_SZ,
```

Comment:
Don' spread them out over multiple lines.
Put it back.

Reply:
Done (although I do believe it is more readable spread out; no worries).